doc/api/report.md

```diff
@@ -8,6 +8,10 @@
 
 <!-- name=report -->
 
+<!-- YAML
+added: REPLACEME
```

Comment:
This should be "changes" with PR url and description. Added is used for newly added APIs.